lld/test/ELF/avr-errors.s

```diff
@@ -0,0 +1,27 @@
+; REQUIRES: avr
+; RUN: llvm-mc -filetype=obj -triple=avr -mcpu=atmega328 %s -o %t.o
+; RUN: not ld.lld %t.o -o %t 2>&1 -Ttext=0x1000 --defsym=callee0=0xf00 --defsym=callee1=0x2048 --defsym=callee2=0x100f | FileCheck %s
+
+.section .LDI,"ax",@progbits
+
+.globl __init
+__init:
+
+; CHECK:     relocation R_AVR_7_PCREL out of range: {{.*}} 'callee0'
```

Comment:
Error messages should start with `error: `

Reply:
Thanks! I have fixed.